### PR TITLE
pull request for 3 commits

### PR DESCRIPTION
**Pull request by dianaelmasri**

### CODE REVIEW

File: OpenHubCodeLocationExtractor.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import csv
+import sys
+import requests
+from HTMLParser import HTMLParser
+
+project = ""
+
+class MyHTMLParser(HTMLParser):
+    foundBlock = False
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:11]: You could cut that.

File: OpenHubCodeLocationExtractor.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    foundCodeLocation = False
+    codeLocation = ""
+
+    tdCounter = 0
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:16]: You could cut that.

File: OpenHubCodeLocationExtractor.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    def handle_starttag(self, tag, attrs):
+        if self.foundBlock and tag == 'td' and len(attrs) > 0 and attrs[0][1] == "col-md-4":
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:19]: Proposal
if tag == 'td' and len(attrs) > 0 and attrs[0][1] == "col-md-4":

File: OpenHubCodeLocationExtractor.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            self.foundCodeLocation = True
+
+        elif tag == 'tr' and len(attrs) > 0 and attrs[0][0] == "class":
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:22]: You could cut that.

File: OpenHubCodeLocationExtractor.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            self.foundBlock = True
+
+
+    def handle_data(self, data):
+        if self.foundCodeLocation and self.codeLocation == "":
+            self.codeLocation = data
+
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:29]: Add these lines here
print(project + "|" + self.codeLocation)
self.foundBlock = False

File: OpenHubCodeLocationExtractor.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+    def handle_endtag(self, tag):
```
Comment (fabiopetrillo) [inline OpenHubCodeLocationExtractor.py:31]: You could cut that method

**Review by fabiopetrillo — CHANGES_REQUESTED** [review]
Great work, and I proposed some changes. :-)

**Comment by dianaelmasri** [comment]
Great I will apply changes

**Pull request closed without merging**